### DISCUSSION TITLE
Survey about open source software development

### DISCUSSION THREAD

**Discussion by shengjingly** [yarnpkg/berry#3814] [General]
Hello, 
I hope this is the right place to ask, but I’m currently looking for participants for my survey that I’m doing for my master thesis. The research deals with the relationship between agile practices and open source software development. Therefore, I’m looking for active contributors of open source projects.
The survey takes only about 3 minutes: [spam]

Thank you in advance for taking the time! I appreciate your help 😊

**Comment by arcanis** [comment]
Please do not spam our repository.
